mandatory/2-function-creation.js

```diff
@@ -4,8 +4,13 @@
     1. the user should be 18 or older
     2. the user must be logged in
 */
```

Comment:
This looks good to me 😄 
Can you think of a way to make this code even shorter? Maybe you can do it without the `if` statement, and just use `return`?